components/pages/Messages/Messages.module.css

```diff
@@ -253,3 +257,100 @@
   width: 100%;
   align-items: center;
 }
+.likeIconContainer {
+  width: max-content;
```

Comment:
I don't think you need this.

Reply:
I actually do, because the like icon has a div and if I don't specify the width it would take the whole width of the section.